### PR TITLE
fix: fix for the comparison/replacement script for sdk md files

### PR DESCRIPTION
**Pull request by samricotta**
Fix for the md file replacement script + updated file from running the script

### CODE REVIEW

**Comment by julienrbrt** [comment]
Can you make the changes @tac0turtle did in https://github.com/cosmos/cosmos-sdk-docs/pull/29 on the action, based on this branch in another PR? Then we can see in that PR the result and if it works properly.

File: sync_script.sh [hunk 1]
```diff
@@ -56,32 +56,30 @@ for version in "${VERSIONS[@]}"; do
...
+          else
+            # Replace the local file with the remote file if differences are found
+            echo "Differences found for $local_file and $remote_file. Replacing $local_file with remote file..."
             cp "./cosmos-sdk/docs/$remote_file" "$local_file"
+            echo "Local file replaced."
           fi
-        done
+        fi
       done
+    done
```
Comment (tac0turtle) [inline sync_script.sh:82]: can we add rm cosmos-sdk after the script runs so we dont push it by accident

File: .github/workflows/copy-md.yml [hunk 2]
```diff
@@ -27,3 +28,8 @@ jobs:
         run: |
           chmod +x sync_script.sh  
           ./sync_script.sh
+          
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        if: always()
+        with:
+          commit_message: sync simapp changes
```
Comment (tac0turtle) [inline .github/workflows/copy-md.yml:35]:
```suggestion
          commit_message: sync documentation changes
```

**Comment by samricotta** [comment]
note: `versioned_docs/version-0.50/build/building-modules/06-keeper.md` needs to be changed in 0.50 release branch

**Comment by julienrbrt** [comment]
I would have expected https://github.com/cosmos/cosmos-sdk/blob/release/v0.50.x/docs/docs/build/packages/README.md to be added https://github.com/cosmos/cosmos-sdk-docs/tree/main/versioned_docs/version-0.50/build/packages but I do not see it here.

**Comment by tac0turtle** [comment]
i dont think the script works for docs changes on main?

**Comment by samricotta** [comment]
Okay i think both your points have been fixed now

**Review by tac0turtle — APPROVED** [review]
tested locally, works thank you. 

main changes are still not updated but seems there is another pr for this

**Pull request merged**